tuf/ngclient/_internal/trusted_metadata_set.py

```diff
@@ -226,6 +226,10 @@ def update_timestamp(self, data: bytes) -> Metadata[Timestamp]:
                     f"New timestamp version {new_timestamp.signed.version} must"
                     f" be >= {self.timestamp.signed.version}"
                 )
+            # If new_timestamp.version == trusted_timestamp.version ignore it.
+            if new_timestamp.signed.version == self.timestamp.signed.version:
```

Comment:
Minor comment nits: 
- Copy-pasting the commented code is not very useful, and actually confusing if it's slightly modified
-  What is 'it'? What is ignored?

What about something like
```suggestion
            # Keep using old timestamp if versions are equal
```

Reply:
Yes, you are right. I did that as I was struggling to find a short way to describe that the new_timestamp version is the same as the old one, but as you showed in the comment that's not needed as the check shows that.